lib/trip_dispatcher.rb

```diff
@@ -27,14 +41,58 @@ def inspect
               #{passengers.count} passengers>"
     end
 
+    #Wave 3: Find driver_by_status
+    def find_driver_status(status)
```

Comment:
The name `find_driver_status` is misleading since it sounds like you are looking for the status of a single driver.

```suggestion
    def find_driver_by_status(status)
```